search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -56,13 +57,25 @@
   [context rfms]
   (map #(rfm->umm-collection context %) rfms))
 
+(defn- refresh-and-get-collections-from-cache
+   "Force the collection cache to be populated and then get all the collections from it."
+   [context]
+   (let [_ (metadata-cache/refresh-collections-metadata-cache-job)
```

Comment:
Actually I made a mistake here. I should call (metadata-cache/refresh-cache) here. not refresh-collections-metadata-cache-job, which is used for scheduler...